### ISSUE TITLE
Outdated dependencies?

### ISSUE THREAD

**Issue by a12e** [OvermindDL1/ecto_interval#3]
Hi,

It seems that the dependencies are too old to allow this module to be installed:

```
Failed to use "phoenix_html" (version 2.13.3) because
  ecto_interval (version 0.2.1) requires ~> 2.12.0
  phoenix_ecto (version 4.0.0) requires ~> 2.9
  scrivener_html (version 1.8.1) requires ~> 2.2
  mix.lock specifies 2.13.3
```

```
Failed to use "postgrex" because
  ecto_interval (version 0.2.1) requires ~> 0.14.0
  ecto_sql (version 3.1.6) requires ~> 0.14.0 or ~> 0.15.0
  mix.lock specifies 0.15.0
```

**Comment by OvermindDL1** [comment]
Published 0.2.2 that allows either 0.14 or 0.15.  :-)

**Comment by a12e** [comment]
Thanks for the incredibly fast update!
